src/getExtensionApi.ts

```diff
@@ -19,7 +19,7 @@ export async function getFunctionsApi(context: IActionContext, installMessage?:
     const funcExtension: AzureExtensionApiProvider | undefined = await getApiExport(funcExtensionId);
 
     if (funcExtension) {
-        return funcExtension.getApi<AzureFunctionsExtensionApi>('^1.3.0');
+        return funcExtension.getApi<AzureFunctionsExtensionApi>('^1.7.0');
```

Comment:
Err, how did we go from 1.3.0 to 1.7.0? I thought we did 1.4.0, decided we didn't need it, reverted the change in 1.5.0.  Aren't we at 1.6.0?